fighters/common/src/function_hooks/change_status.rs

```diff
@@ -53,10 +55,27 @@ unsafe fn change_status_request_from_script_hook(boma: &mut BattleObjectModuleAc
     let mut clear_buffer = arg3;
 
     if boma.is_fighter() {
+        match utils::game_modes::get_custom_mode() {
```

Comment:
Instead of putting this in a status change request hook, please put this in is_enable_transition_term instead

Reply:
There are places where we transition into airdodge (like wavedash) where we don't check transition term, I think this is fine.

Reply:
in that case we really should be refactoring that so that it does use the system, but I won't push too hard for it now